### ISSUE TITLE
Idea: Set default WorldBalanceGroup in config

### ISSUE THREAD

**Issue by Rocologo** [greatman/craftconomy3#30]
Hi Greatman. 

I have a idea for you. It would be very nice if you could set the default "WorldBalanceGroup" in config.yml

This is the reason: I have a multiworld server. I have three WorldGroups: One for Survival worlds (default), one for Creative worlds (creativeBalance) and one for my ASkyBlock world (askyblockBalance). The creative and survival worlds can obviously not share economy. The problem is that some users spawn in a survival world and some users spawn in a creative world - depending on their rank. This means that in some cases their initial mount of money will end up in different worldgroups, depending on where they login after first install of Craftconomy. I want the money to end up in the survivalBalance (default). They don't need the money in the creative worlds.

/Rocologo

**Comment by greatman** [comment]
So basicly a config flag that makes Craftconomy deposit in the wallet you want instead of just in the default one?

**Comment by Rocologo** [comment]
Correct, I want the initial money the users get, to be deposited in the 
Survival Wallet and not in default which is a Creative world. For 
backward compability you can set the config flag to default, then I just 
change this. :-)
